docs/building/netcoreapp3.1-instructions.md

```diff
@@ -1,14 +1,9 @@
-ML.NET source code files build for .NET Core 3.1 and .NET Standard 2.0. However, ML.NET test files only build on one target framework below at a time:
-- .NET Core 2.1
-- .NET Core 3.1
-- .NET Framework 4.6.1
+ML.NET source and test code files build for .NET Core 3.1 and NET Framework 4.6.1.
 
-To run tests on .NET Core 3.1, you need to do a few manual steps.
+If you want to run command line tests on only 1 of the frameworks or on a different framework, do this:
+ add `/p:TestTargetFramework=<Framework Names>` to the command line build.
```

Comment:
Maybe this doesn't need to happen on this first PR, but it would be best/nice if we supported the "normal" way of `dotnet test -f <FRAMEWORK>`

Reply:
This does work on individual projects just fine. These instructions are for the `build.cmd` script itself. Is there a better way to handle this with a build script and not just directly with `dotnet test`?

Reply:
I almost wonder if this page is still needed going forward... thoughts on removing it?

Reply:
I think we are good to remove it. since 3.1 is now default we don't need separate instructions.